packages/hardhat-ignition-core/test-integrations/test-execution-result-fixtures.ts

```diff
@@ -98,7 +98,11 @@ describe("execution-result-fixture tests", function () {
     for (const contractName of contractNames) {
       const artifact = staticCallResultFixturesArtifacts[contractName];
 
-      for (const abiItem of artifact.abi) {
+      for (const abiItem of artifact.abi.filter(
+        // TODO: bring this test back once EDR has updated the invalid
+        // error message error message - I said what I said
+        (abi) => abi.name !== "revertWithInvalidErrorMessage"
```

Comment:
I initially removed the method from the Solidity contract - but that implied fixing up bytecode assertions across the test suite.